content/docs/quickstart/azure/destroy-stack.md

```diff
@@ -44,6 +44,8 @@ Duration: 53s
 
 To delete the stack itself, run `pulumi stack rm`.
 
+{{< destroy-warning >}}
```

Comment:
Hm. So my two cents on this is that since we're explaining how to do this (indeed this page is all about destruction), adding a warning about doing it feels misplaced. I do think it'd be helpful, though, to lend a few words to the line above it about what `stack rm` does (along with a link to its docs), just as a matter of course -- like, `To delete the stack itself from pulumi.com, along with all of its history, run pulumi stack rm`.